tests/python/gaia-ui-tests/gaiatest/tests/graphics/test_browser_play_video.py

```diff
@@ -0,0 +1,70 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from marionette_driver import Wait
+
+from gaiatest.apps.search.app import Search
+from gaiatest.apps.search.regions.html5_player import HTML5Player
+from gaiatest.apps.homescreen.regions.permission_dialog import PermissionDialog
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+
+class TestVideo(GaiaImageCompareTestCase):
+
+    def setUp(self):
+        GaiaImageCompareTestCase.setUp(self)
+        self.connect_to_local_area_network()
+        self.video_URL = self.marionette.absolute_url('VID_0001.ogg')
+
+    def test_play_video(self):
+        """Confirm video playback
+
+        https://moztrap.mozilla.org/manage/case/6073/
+        """
+        search = Search(self.marionette)
+        search.launch()
+        browser = search.go_to_url(self.video_URL)
+        browser.wait_for_page_to_load(180)
+        browser.switch_to_content()
+        player = HTML5Player(self.marionette)
+        player.wait_for_video_loaded()
+
+        # Wait until the video plays to the end
+        play_time = 20.0
+        self.wait_for_condition(
+            lambda m: player.current_timestamp == play_time,
+            timeout=30,
+            message='Video did not reach the end')
+        self.take_screenshot(True, browser)
+
+        # Tap UI buttons. player must be reinstantiated each time after screenshot is called, because it loses context
+        player = HTML5Player(self.marionette)
+        player.tap_mute()
+        self.take_screenshot(True, browser)
+        player = HTML5Player(self.marionette)
```

Comment:
each time the screenshot is taken, it goes to the system frame to detect the status bar.  it loses the context to the player instance even if we return to the browser application frame. Thus it is necessary to re-reference HTML5Player after the screenshot is taken.